logrotation/startup/entrypoint_cron.py

```diff
@@ -0,0 +1,73 @@
+# /usr/bin/python3
+
+import argparse
+import sys
+from cortx.utils.conf_store import Conf
+from cortx.utils.process import SimpleProcess
+import os
+
+sys.path.append('/opt/cortx/component/test')
+from logger import start_logging
+
+
+def get_local(config_url):
```

Comment:
No need to create a function for one line code. 
Two issues
1. Use Conf.get directly in the code in the beginning. 
2. Use Conf.load in the beginning (execute only once).